### DISCUSSION TITLE
Looking for a feature - Auto preview the search record (if it is a file, site etc) if the mouse over onto the search record.

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2334] [Ideas]
Hi team,

I'm using PnP modern search at my QA environment, Were I found that the search records are not enabled or activated by default with auto preview when an end user will mouse over to any search record. 
We saw the auto preview functionality  at classic search and would like to implement at PnP Modern results area.  Currently PnP modern search results webpart looking for a click (PnP-Filepreview) method on preview image and could you please help or guide us on how tto implementing an auto preview (when mouse over the preview image).

I have checked the code with cloning the code GitHub repository but no luck. (because I'm yet to learning development).

thank you

Regards,
gangadhar

**Comment by wobba** [comment]
If you want to preview on hover you need to change the code. Adding an option to the https://microsoft-search.github.io/pnp-modern-search/extensibility/web_components_list/#pnp-filepreview component seems like a good choice. From an accessibility point of view this is not good behavior though.

**Comment by GangadharMFord** [comment]
thank you @wobba, for your response and to be honest, I'm not a developer but I love learning the development. Kindly help me on where and what to change in filepreview it will greatly helps me on going out from this tough situation.

thank you again,
Gangadhar

**Reply by wobba** [reply]
I'll leave it here as an enhancement for a dev to pick up on. Taking the time to figure out what to change is the same as doing it more or less :)

**Reply by BVSanjay** [reply]
Is this feature available, preview on hover of documents card?

**Reply by kasperbolarsen** [reply]
Take a look at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/ , as that feature is available for the Card and List layouts :-)
